src/map/map.ts

```diff
@@ -346,6 +342,14 @@ export class PropertiesMap {
         this._settingsModal.remove();
     }
 
+    public get active() {
+      if (this._selected.size > 0) {
+        if (this._active === undefined) {
+        this.active = this._selected.keys().next().value;
+        return this._selected.keys().next().value;
+        } else { return this._active; }
```

Comment:
minor nitpick: I prefer having the statements for the else case on a separate line, I feel it is easier to parse quickly when looking at the code